apps/docs/content/styled-system/concepts/styling.mdx

```diff
@@ -42,6 +42,10 @@ It will also help you to search for these usages when you want to update them, o
 
 <CodeOnlyExample src="hopper-provider/docs/styling/escape-hatch" />
 
+<AI>
+⚠️ **CRITICAL**: Props with a "Tokens Group" of "none" should be used directly. These props do not support design tokens, so you should not use the `UNSAFE_` prefix or attempt to apply tokens to them.
```

Comment:
Is there a way to phrase it where we don't reference the column name directly? since this is not in the same file as the Token Group, if we rename the column, we will 100% forget to update this